### DISCUSSION TITLE
Cname and wildcard domains

### DISCUSSION THREAD

**Discussion by b2rsp** [vercel/community#335] [Show & Tell]
Hi,

I have a project that allows small teachers and art schools to have its website that is hosted in Vercel and using Nextjs. So I have enabled the wildcard domains and therefore I have something like: schooName1.app.com  and schoolName2.app.com etc.

My question is if we want to allow the teachers to have a CNAME redirect, for example: 
CNAME schoolName.com to schoolName1.app.com 
How could I do that because I am but clueless at the moment... Any help would be deeply appreciated.

**Comment by leerob** [comment]
Good news! We have a guide for this: https://vercel.com/guides/nextjs-multi-tenant-application
